### ISSUE TITLE
regression: cannot edit or save anymore

### ISSUE THREAD

**Issue by bertsky** [OCR4all/LAREX#303]
```
org.springframework.web.servlet.handler.AbstractHandlerExceptionResolver.resolveException Resolved [
  org.springframework.http.converter.HttpMessageNotReadableException: 
    JSON parse error: Cannot deserialize instance of `org.primaresearch.dla.page.layout.physical.text.impl.Glyph` out of START_ARRAY token; 
  nested exception is com.fasterxml.jackson.databind.exc.MismatchedInputException: 
    Cannot deserialize instance of `org.primaresearch.dla.page.layout.physical.text.impl.Glyph` out of START_ARRAY token at [Source: (PushbackInputStream); line: 1, column: 1676]
 (through reference chain: de.uniwue.web.communication.ExportRequest["segmentation"]
  ->de.uniwue.web.model.PageAnnotations["segments"]
  ->java.util.LinkedHashMap["OCR-D-SEG-BLOCK-tesseract_lieferschein_3Dmensionals.0000_region0009"]
  ->de.uniwue.web.model.Region["textlines"]
  ->java.util.LinkedHashMap["OCR-D-SEG-BLOCK-tesseract_lieferschein_3Dmensionals.0000_region0009_line0005"]
  ->de.uniwue.web.model.TextLine["words"]
  ->java.util.ArrayList[0]->de.uniwue.web.model.Word["glyphs"]
  ->java.util.ArrayList[0]->de.uniwue.web.model.GlyphContainer["glyphVariants"])]
```

It happened between 7ce4537d7297cbc19f595bfd2ef07dc7ab72d501..31bccf30995eba4c4e5071188421215770ca8a51, which would implicate #294.

I can see some changes to Glyph variant handling there (but do not understand them).

**Comment by bertsky** [comment]
Here's an XML file example [regression.zip](https://github.com/OCR4all/LAREX/files/7906019/regression.zip)

**Comment by maxnth** [comment]
Should be fixed by #305 thanks to @chaddy314

**Comment by bertsky** [comment]
It is – thanks @chaddy314 for your fast reaction, much appreciated!
